src/diffusers/models/attention.py

```diff
@@ -429,6 +518,38 @@ def forward(self, x: torch.Tensor, timestep: torch.Tensor) -> torch.Tensor:
         return x
 
 
+class AdaLayerNormContinuous(nn.Module):
```

Comment:
Similar to AdaLayerNorm except that the passed conditioning is not discrete and so does not require and embedding layer